server/publicapi/features/items.feature

```diff
@@ -3,10 +3,14 @@ Feature: Public content
     Scenario: Read an item
         Given "items"
         """
-        [{"_id": "tag:example.com,0000:newsml_BRE9A605", "mimetype": "text/plain"}]
+        [{
+            "_id": "tag:example.com,0000:newsml_BRE9A605",
+            "headline": "Breaking news in Timbuktu 123",
+            "mimetype": "text/plain"
+        }]
         """
         When we get "/items/tag:example.com,0000:newsml_BRE9A605"
         Then we get existing resource
         """
-        {"_id": "tag:example.com,0000:newsml_BRE9A605", "mimetype": "text/plain"}
+        {"headline": "Breaking news in Timbuktu 123", "mimetype": "text/plain"}
```

Comment:
This test does not verify that _id is not returned. You should create a step that given fields do not exist in the returned document.